config/main.yaml

```diff
@@ -1,14 +1,14 @@
 defaults:
   - _self_
   - env: grid
-  - gflownet: flowmatch
+  - gflownet: trajectorybalance
   - policy: mlp_${gflownet}
-  - proxy: corners
+  - proxy: uniform
   - logger: wandb
   - user: default
 
 # Device
-device: cuda
+device: cpu
```

Comment:
I would suggest adding somewhere (e.g. README, very early on in description of how to run code) information that CUDA is NOT the default - I understand why it's the case (CPU being faster on small tasks), but I think the expected thing is to use GPU by default if available.

Reply:
Ok, added: https://github.com/alexhernandezgarcia/gflownet/pull/315/commits/00cb7e129dadd283619b17fa8d59877317a97fb3